app-emulation/acbuild/acbuild-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2015 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+CROS_WORKON_PROJECT="appc/acbuild"
+CROS_WORKON_REPO="git://github.com"
+CROS_WORKON_LOCALNAME="appc-acbuild"
+inherit cros-workon
+
+if [[ "${PV}" == 9999 ]]; then
+    KEYWORDS="~amd64 ~arm64"
+else
+    CROS_WORKON_COMMIT="e49e7de27443374a5356b7ccd9d9d7691fb5089f" # v0.2.2
+    KEYWORDS="amd64 arm64"
+fi
+
+DESCRIPTION="A build tool for ACIs"
+HOMEPAGE="https://github.com/appc/acbuild"
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.4.3"
```

Comment:
Yes, won't be moving to 1.5 until we get docker 1.10
On Feb 1, 2016 7:22 PM, "Derek Gonyeo" <notifications@github.com> wrote:

> In app-emulation/acbuild/acbuild-9999.ebuild
> <https://github.com/coreos/coreos-overlay/pull/1743#discussion_r51518299>:
>
> > +if [[ "${PV}" == 9999 ]]; then
> > +    KEYWORDS="~amd64 ~arm64"
> > +else
> > +    CROS_WORKON_COMMIT="e49e7de27443374a5356b7ccd9d9d7691fb5089f" # v0.2.2
> > +    KEYWORDS="amd64 arm64"
> > +fi
> > +
> > +DESCRIPTION="A build tool for ACIs"
> > +HOMEPAGE="https://github.com/appc/acbuild"
> > +SRC_URI=""
> > +
> > +LICENSE="Apache-2.0"
> > +SLOT="0"
> > +IUSE=""
> > +
> > +DEPEND=">=dev-lang/go-1.4.3"
>
> Is CoreOS still on go 1.4? It's getting kinda old by now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/coreos-overlay/pull/1743/files#r51518299>.
>